pallet/dex/src/lib.rs

```diff
@@ -512,6 +534,75 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		#[pallet::weight({5})]
+		#[pallet::call_index(5)]
+		pub fn deposit_native(
```

Comment:
We have an issue here. Let's not mix-up wrapped ggx (asset) with native ggx.
I see potential scenario where user can't get funds.

Imagine some user deposited wrapped ggx into.
Another user deposited native ggx. They both have balance as asset ggx.

But then first user retrieved as native, and the one that put native can't because balance on dex is empty.

Let's create dummy id for native coins. I use for that substrate address prefix on UI. 

@xcthulhu wdyt?

Reply:
We have decided to stick to reserved id for now...